Algorithms/JavaScript/hamming-weight.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Write a function that takes an unsigned integer and returns the number of ’1' bits it has (also known as the Hamming weight).
+ * @param {number} n - a positive integer
+ * @return {number}
+ */
+var hammingWeight = function(n) {
+  return n.toString(2).split("1").length - 1
```

Comment:
This can also be done using division/modulus, or (way cooler imo) bitwise operators.